e107_plugins/download/download_shortcodes.php

```diff
@@ -918,9 +918,10 @@ function sc_download_cat_search()
 			
 		$text .= '
               <span class="input-group-btn">
-              <button class="btn btn-default" type="submit" name="s"  value="1">
-              <span class="glyphicon glyphicon-search"></span>
-             </button>
+              <button class="btn btn-default" type="submit" name="s"  value="1">';
+              
+              $text .= '<span class="'.((deftrue('BOOTSTRAP') && deftrue('FONTAWESOME') ? "fa fa" : "glyphicon glyphicon").'-search"></span>';
```

Comment:
Candidate to recode using the toGlyph function???
Maybe like this:
`$text .= $tp->toGlyph('glyphicon-search');`